src/runtime_src/core/pcie/driver/linux/xocl/xocl_drm.h

```diff
@@ -25,6 +25,8 @@
 typedef void (*xocl_execbuf_callback)(unsigned long data, int error);
 
 #define IS_HOST_MEM(m_tag)	(!strncmp(m_tag, "HOST[0]", 7))
+#define IS_PLRAM(m_tag)		(!strncmp(m_tag, "PLRAM[0]", 8))
```

Comment:
we may have more than one PLRAM, right? should we just compare "PLRAM["?

Reply:
That was my initial thought. But I am not sure why we are using HOST[0]? Maybe HOST[1] means differently?

Reply:
XRT currently can only support one slave port, which is "HOST[0]". "HOST[x]" is not expected...

Reply:
done